client/src/actions/inventory.js

```diff
@@ -5,7 +5,9 @@ const API_URL = `${window.location.origin}/api/v1/inventory_items`;
 
 const headers = {};
 headers['Authorization'] = `Bearer ${cookie.load('token')}`;
-headers['workspace-id'] = cookie.load('current_workspace').id;
+if (cookie.load('current_workspace')) {
```

Comment:
why do we need this here?
Inventory indeed is workspace related, so is it possible to not having workspace id in the cookies?

Reply:
expose :items, -> { current_workspace.inventory_items }
inventory is workspace related because each workspace has its own inventory and also its own counterparties